.circleci/verbatim-sources/job-specs/pytorch-job-specs.yml

```diff
@@ -328,6 +328,11 @@ jobs:
       - checkout
       - attach_workspace:
           at: c:/users/circleci/workspace
+      - run:
+          name: Install VS2019
+          no_output_timeout: 30m
```

Comment:
Does this mean that we have to wait about 30m to install VS 2019? OMG

Reply:
I don't think so.
we never wait 1h to install miniconda
https://github.com/pytorch/pytorch/blob/22a34bcf4e5eaa348f0117c414c3dd760ec64b13/.circleci/verbatim-sources/nightly-binary-build-defaults.yml#L39

Reply:
So could you tell me how long it takes to configure VS2019? I don't think it is sensible to wait 30 min on environment setup.

Reply:
Looks like it takes <2m to install VS2019, which is pretty fast.